awx/ui/test/e2e/fixtures.js

```diff
@@ -306,17 +306,28 @@ const getUpdatedProject = (namespace = session) => {
  * @param [namespace] - Name prefix for associated dependencies.
  * @param [playbook] - Playbook for the job template.
  * @param [name] - Unique name prefix for the job template.
+ * @param [updateProject] - Choose whether to sync the project with its repository.
  * */
 const getJobTemplate = (
     namespace = session,
     playbook = 'hello_world.yml',
-    name = `${namespace}-job-template`
+    name = `${namespace}-job-template`,
+    updateProject = true
 ) => {
-    const promises = [
-        getInventory(namespace),
-        getAdminMachineCredential(namespace),
-        getUpdatedProject(namespace)
-    ];
+    let promises;
+    if (updateProject) {
```

Comment:
I think if you did something like (kinda of pseudo-code):

```
promises  = [
    getInventory(namespace),
    getAdminMachineCredential(namespace),
]

if updateProject {
    promises.append(getUpdatedProject(namespace))
} else {
    promises.append(getProject(namespace))
}
```

Would make it easier to read, I had to go line by line to figure that only the last item is different.

Reply:
Great call, I can make this adjustment.